bin/common.ml

```diff
@@ -530,6 +533,12 @@ let term =
           ~doc:
             "Instead of terminating build after completion, wait continuously \
              for file changes.")
+  and+ instrument_with =
+    Arg.(
+      value & opt_all string []
+      & info [ "instrument-with" ]
+          ~doc:
+            "Enable instrumentation by the given backends.")
```

Comment:
The doc looks like this:

```
 --instrument-with=VAL
     Enable instrumentation by the given backends.
```

So I'm thinking we should probably remove the `s` and mention that the option can be repeated. WDYT? Otherwise it looks like `VAL` is a list of backend. We could also make `VAL` be a coma-separated list, we do that for `--only-packages` for instance.

Could you also replace `VAL` by `BACKEND` (or `BACKENDS`)?

Reply:
Amended as suggested (with one flag per backend).